deploy/olm-catalog/unifiedpush-operator/0.2.0/unifiedpush-operator.v0.2.0.clusterserviceversion.yaml

```diff
@@ -25,6 +25,11 @@ spec:
       version: v1alpha1
       displayName: Android Variant
       description: Represents a UPS Android Variant
+    - kind: IOSTokenVariant
+      name: iostokenvariants.push.aerogear.org
+      version: v1alpha1
+      displayName: iOS Variant
+      description: Represents a UPS iOS Variant      
```

Comment:
Remove this, it's the CSV for an older version. When we release a new version this variant should then show up.